__init__.py

```diff
@@ -17,34 +17,32 @@
 
 import gettext
 import html
-import json
 import os
 import time
+
+from pyradios import RadioBrowser
 from shutil import which
 from signal import SIGTERM
 from subprocess import Popen
-from urllib.request import Request, urlopen
-from urllib.error import URLError
 
 from pext_base import ModuleBase
 from pext_helpers import Action, SelectionType
 
 
 class Module(ModuleBase):
-    def init(self, settings, q):
+    def __init__(self, settings, q):
```

Comment:
Please don't rename this to `__init__`. It would normally be correct Python, but `__init__` is called when the module is imported. `init` is a function Pext calls when the module is opened. Renaming this function will probably break things like module reloading.